controller/src/main/java/io/pravega/controller/server/rest/ModelHelper.java

```diff
@@ -162,6 +195,29 @@ public static final StreamProperty encodeStreamResponse(String scope, String str
                     }
                     retentionConfig.setTimeBasedRetention(timeRetention.days(days).hours(hours).minutes(minutes));
                     break;
+                case CONSUMPTION:
+                    retentionConfig.setType(RetentionConfig.TypeEnum.CONSUMPTION);
+                    ConsumptionLimits consumptionLimits = new ConsumptionLimits();
+                    RetentionPolicy.ConsumptionLimits cl = streamConfiguration.getRetentionPolicy().getConsumptionLimits();
+                    int divider;
+                    switch (cl.getType()) {
+                        case SIZE_BYTES:
+                            consumptionLimits.setType(ConsumptionLimits.TypeEnum.SIZE_MB);
+                            divider = BYTES_TO_MB;
+                            break;
+                        case TIME_MILLIS:
+                            divider = MILLIS_TO_MINUTES;
+                            consumptionLimits.setType(ConsumptionLimits.TypeEnum.TIME_MINUTES);
+                            break;
+                        default:
```

Comment:
Add NONE here to handle the case where user does not want to have a fallback policy.